### PR TITLE
Deps refact

### PR DESCRIPTION
**Pull request by nastasi-oq**
More consistent and smart dependencies scheme.
Test are running here: https://ci.openquake.org/job/zdevel_oq-platform/172/

### CODE REVIEW

**Comment by daniviga** [comment]
LGTM

File: openquakeplatform/setup.py [hunk 1]
```diff
@@ -74,16 +74,16 @@ def read(*rnames):
         "django-activity-stream==0.4.5beta1",
         "django-announcements==1.0.2",
         "django-appconf==0.6",
-        "django-chained-multi-checkboxes==0.4",
-        "django-chained-selectbox==0.2.1",
+        "django-chained-multi-checkboxes >= 0.4, < 0.5",
```
Comment (daniviga) [inline openquakeplatform/setup.py]: In this way, with an old installation with the old tagged 0.4 `django-chained-multi-checkboxes` will not be updated to the new version

```
Requirement already satisfied (use --upgrade to upgrade): django-chained-multi-checkboxes>=0.4,<0.5 in /home/ubuntu/platform-env/lib/python2.7/site-packages (from openquakeplatform==1.3.0-gitba6ef10)
```

File: verifier.sh [hunk 2]
```diff
@@ -220,6 +220,8 @@ usage () {
     echo "    $0 devtest <branch-name>"
     echo "                                                 put oq-platform sources in a lxc,"
     echo "                                                 setup environment and run development tests."
+    echo "    $0 prodtest <branche-name>"
```
Comment (ptormene) [inline verifier.sh]: Typo `branche -> branch`

**Comment by ptormene** [comment]
LGTM

**Pull request merged**